atomic_reactor/plugins/pre_pin_operator_digest.py

```diff
@@ -449,6 +453,45 @@ def _get_replacement_pullspecs_OSBS_resolution(self, pullspecs):
 
         return replacements
 
+    def _replace_other_metadata_from_csv_modifications(self, operator_manifest):
+        """
+        WILL add/append/update/replace metadata in the operator manifest
+        passed in
+
+        :param operator_manifest: an OperatorManifest, as created by `from_directory()`
+        :type operator_manifest: instance of OperatorManifest
+        :return: None
+        """
+
+        def do_action(operator_manifest, action):
+            mods = self._fetch_operator_csv_modifications()
+            if (mods.get(action)):
+                # Loop over mod data and apply action
+                for action_field in mods[action]:
+                    for item in mods[action][action_field]:
+                        if action == 'append':
+                            if isinstance(mods[action][action_field][item], list):
+                                # Append to existing list
+                                if operator_manifest.csv.data[action_field].get(item):
+                                    for value in mods[action][action_field][item]:
+                                        print(value)
+                                        operator_manifest.csv.data[action_field][item].append(value)
+                                else:  # Add list - it doesn't exist in target CSV
+                                    operator_manifest.csv.data[action_field][item] = mods[action][action_field][item]
```

Comment:
All modifications to CSV should be done via OperatorManifest object, don't bypass it, create methods there.

Reply:
https://en.wikipedia.org/wiki/Separation_of_concerns